src/components/expanded-state/SupportedCountriesExpandedState.js

```diff
@@ -58,7 +59,7 @@ const SupportCountriesExpandedState = () => {
         wiggleFactor={0}
       >
         <AssetPanel>
-          <Centered css={padding(19, 30, 24)} direction="column">
+          <Centered css={padding.object(19, 30, 24)} direction="column">
```

Comment:
extract

Reply:
don't you memo this anyway?

Reply:
yes, it's memoized, but I was hoping for actually dripping deep memo at some point and instead making sure we can rely on shallow equal instead which is faster